.gitignore

```diff
@@ -21,6 +21,8 @@ lib/cartopy/tests/mpl/output/
 docs/source/cartopy_outline.rst
 docs/source/examples
 docs/source/gallery.rst
+docs/source/gallery
+docs/source/api
```

Comment:
this api section is auto-generated by sphinx-gallery and can be used to build mini-galleries for specific functions / classes if desired